netwrk/netwrk.go

```diff
@@ -127,6 +134,11 @@ func Call(
 		request.Header.Set("Content-Type", "application/json")
 	}
 
+	for _, apiKey := range additionalAPIKeys {
+		if apiKey != (AdditionalAPIKey{}) {
+			request.Header.Set(apiKey.Key, apiKey.Value)
+		}
+	}
```

Comment:
i might consider minimizing such code that is so specific to the manifest provider service from ending up in a package wide helper like `Call`. an example of an alternative approach could be having the additional param be `headers` or `additionalHeaders` that would be of type `map[string]string` or something like that. then you can keep the additional api keys business within the manifest provider service code instead (client.go). additionally could remove the `AdditionalApiKey` struct from `netwrk.go` as well as `client/api.go` and opt to include that type in the already created `manifestProvider.go` file as it only has relevance to that service. just some initial thoughts i haven on the subject